jobs/video_jobs/maya_submit.sh.erb

```diff
@@ -19,7 +19,7 @@ CMD="singularity exec $MAYA_IMG Render"
 RENDERER="<%= attributes['renderer'] %>"
```

Comment:
use the getter "renderer" not the attributes hash

Reply:
Done.